fieldservice/models/res_config_settings.py

```diff
@@ -84,6 +86,14 @@ def _onchange_module_fieldservice_stock(self):
         if self.module_fieldservice_stock:
             self.group_stock_production_lot = True
             self.group_stock_request_order = True
+            if self.module_fieldservice_sale:
+                self.module_fieldservice_sale_stock = True
+
+    @api.onchange('module_fieldservice_sale')
+    def _onchange_module_fieldservice_sale(self):
+        if self.module_fieldservice_sale and \
+           self.module_fieldservice_stock:
+            self.module_fieldservice_sale_stock = True
```

Comment:
Use auto_install = True and depends in fieldservice_sale_stock/\_\_manifest\_\_.py instead of the _onchange.

Reply:
Thank you! Updated in latest commit